### PR TITLE
Added Profile Page for logged in users

### PR DESCRIPTION
**Pull request by sarakvaska**

### CODE REVIEW

File: src/main/webapp/WEB-INF/view/activityfeed.jsp [hunk 1]
```diff
@@ -14,7 +14,11 @@
     <% } else{ %>
       <a href="/login">Login</a>
     <% } %>
-    <a href="/about.jsp">About</a>
+    <% if(request.getSession().getAttribute("user") != null){ %>
+      <a href="/profile">Profile</a>
+    <% } else{ %>
```
Comment (aljonpineda) [inline src/main/webapp/WEB-INF/view/activityfeed.jsp]: I think that even if the user is logged in, they should still be able to access the about page.

File: src/main/webapp/WEB-INF/web.xml [hunk 2]
```diff
@@ -76,4 +76,14 @@
...
   </servlet-mapping>
 
+  <servlet>
+    <servlet-name>ProfileServlet</servlet-name>
+    <servlet-class>codeu.controller.ProfileServlet</servlet-class>
+  </servlet>
+
+  <servlet-mapping>
+    <servlet-name>ProfileServlet</servlet-name>
+    <url-pattern>/profile</url-pattern>
```
Comment (aljonpineda) [inline src/main/webapp/WEB-INF/web.xml]: I think the files ProfileServlet.java and profile.jsp are missing in this pull request.

Comment (sarakvaska) [reply src/main/webapp/WEB-INF/web.xml]: You're so right. I'm going to create a new pull request

File: src/main/webapp/WEB-INF/view/activityfeed.jsp (hunk 1 above)
Comment (sarakvaska) [reply src/main/webapp/WEB-INF/view/activityfeed.jsp]: Ok, will change now

**Pull request closed without merging**